CHANGES.rst

```diff
@@ -1,18 +1,3 @@
-0.6.5 (2024-08-12)
-------------------
-
-Bug Fixes
-^^^^^^^^^
-
-- Multiple Python versions are handled correctly (#1444)
-- JSONC fixes (#1426)
-
-Other Changes and Additions
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-- New documentation design
-
-
```

Comment:
Did you intend to remove this?

Reply:
Yup, since I've added back `towncrier` support and added the fragments so PRs can be merged out of order without having conflicts in the changelog.